frontend/src/scenes/insights/filters/BreakdownFilter/TaxonomicBreakdownButton.tsx

```diff
@@ -30,6 +30,11 @@ export function TaxonomicBreakdownButton({
                 sideIcon={null}
                 disabledReason={disabledReason}
                 size={size}
+                tooltipDocLink={
+                    taxonomicBreakdownType === TaxonomicFilterGroupType.CohortsWithAllUsers
+                        ? 'https://posthog.com/docs/product-analytics/trends/breakdowns#cohorts-and-breakdowns'
+                        : 'https://posthog.com/docs/product-analytics/trends/breakdowns'
+                }
```

Comment:
we could shorten this a little with a template literal.

```suggestion
                tooltipDocLink={`https://posthog.com/docs/product-analytics/trends/breakdowns${
                taxonomicBreakdownType === TaxonomicFilterGroupType.CohortsWithAllUsers
                        ? '#cohorts-and-breakdowns' : ''
                }`}
```